dev-python/asynctest/files/disable-failing-tests.patch

```diff
@@ -0,0 +1,49 @@
+diff --git a/test/test_selector.py b/test/test_selector.py
+index 200fb18..5698e43 100644
+--- a/test/test_selector.py
++++ b/test/test_selector.py
+@@ -286,25 +286,25 @@ class Test_fail_on_active_selector_callbacks(Selector_TestCase):
+ 
+         TestCase().debug()
+ 
+-    def test_events_watched_outside_test_are_ignored(self):
```

Comment:
Why does it fail? Also, don't use a big patch when tiny `sed` can handle it.

Reply:
hi, @mgorny !

I did the best I could at figuring out why it fails. I am not a python dev, so it was a challenge for me :)

I suspect portage sandboxing does not allow to register a new listener for i/o event on a full scale asyncio loop - this only test that fails is the only test that does not use standard procedures for creating a loop.
I narrowed down patching the source to just 1 line and execute it via `sed`.

I examined the source code of python related to this piece and was able to follow it until it goes into actually registering a listener in epoll. I tried to build the package without `sandbox` feature, but it was still failing though. Another reason that makes me think it must be something about portage sandboxing is the fact that nobody else reports similar kind of exception in the upstream issue queue.